Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/MainFragment.kt

```diff
@@ -74,6 +77,17 @@ class MainFragment : BaseFragment() {
         tracingViewModel.refreshActiveTracingDaysInRetentionPeriod()
         TimerHelper.checkManualKeyRetrievalTimer()
         submissionViewModel.refreshDeviceUIState()
+        main_scrollview.sendAccessibilityEvent(AccessibilityEvent.TYPE_VIEW_FOCUSED)
+    }
+
+    override fun onStart() {
+        super.onStart()
+        main_scrollview.sendAccessibilityEvent(AccessibilityEvent.TYPE_VIEW_FOCUSED)
+    }
+
+    private fun setContentDescription() {
+        main_header_share.setContentDescription("Teilen")
```

Comment:
please don't use strings directly, always put them in the strings.xml.

relevant for all strings